data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -107,13 +152,13 @@ _ "This attack uses 6 attacks"#enddef
     [event]
         name = new turn
         first_time_only = no
-        id = song_verse_new_turn_{X}
+        id = song_verse_new_turn_{MAX}
 
         [modify_unit]
             [filter]
                 side = $side_number
```

Comment:
Use of `side_number` within `new_turn`, instead of within `side_turn`. I think it's set to 1 at that point, which would explain why it isn't causing an edge condition.

Reply:
I will implement that.